content-api/requirements.txt

```diff
@@ -5,12 +5,12 @@ gunicorn==20.1.0
 google.auth==2.16.0
 requests==2.28.2
 # opentelemetry libraries
-opentelemetry-api==1.17.0
-opentelemetry-sdk==1.17.0
-opentelemetry-distro==0.38b0
-opentelemetry-exporter-gcp-trace==1.4.0
-opentelemetry-instrumentation==0.38b0
-opentelemetry-instrumentation-flask==0.38b0
+opentelemetry-api==1.18.0
+opentelemetry-sdk==1.18.0
+opentelemetry-distro==0.39b0
+opentelemetry-exporter-gcp-trace==1.5.0
+opentelemetry-instrumentation==0.39b0
+opentelemetry-instrumentation-flask==0.39b0
 opentelemetry-resourcedetector-gcp==1.3.0a0
```

Comment:
```suggestion
opentelemetry-resourcedetector-gcp==1.5.0a0
```

Reply:
Reason: `Error: ResolutionImpossible  opentelemetry-exporter-gcp-trace 1.5.0 depends on opentelemetry-resourcedetector-gcp==1.* and >=1.5.0dev0
Step #0 - "Install Dependencies":     The user requested opentelemetry-resourcedetector-gcp==1.3.0a0` https://pantheon.corp.google.com/cloud-build/builds;region=global/826efd68-ca28-490d-a696-060f0954b093?project=emblem-ci-ops